app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
```

Comment:
The use of a class-level dictionary `people` to store `Person` instances by name assumes that names are unique. This can lead to overwriting instances if two people share the same name. Consider using a unique identifier for each person to avoid this issue.